### ISSUE TITLE
syncthing changes dont manifest when varnish is serving a cached page

### ISSUE THREAD

**Issue by pirog** [kalabox/kalabox#748]
put this on the list of biggest DUHz of all time

i think the best way to get around this for now is to set the NO_CACHE cookie in settings.php

```php
if (PANTHEON_ENVIRONMENT === 'kalabox') {
  setcookie("NO_CACHE", 'kalabox');
}
```

probably need to restart the app as well.

**Comment by pirog** [comment]
In the future i think maybe we want to set some logic so the user can toggle varnish on and off easily.

maybe we can get varnish to pass through when an env var is set?

**Comment by labboy0276** [comment]
For D8, there are 3 settings files now.  settings.php, settings.pantheon.php and settings.local.php.

The local gets loaded last.  I would suggest adding any kalabox type thingz and stuff to that file as it seems the best practices thing to do.

**Comment by pirog** [comment]
This should be resolved in the 0.11.0 release on Friday
